### PR TITLE
Webapp

### PR DESCRIPTION
**Pull request by himanshu1603**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

***********EDITING AN EXISTING TOPIC OR COLLECTION************

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this topic should be curated:

************CURATING A NEW COLLECTION************

- [ ] I've formatted my changes as a new folder directory, named for the collection as it appears in the URL on GitHub (e.g. `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this collection should be curated:

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/webapp/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+aliases: web-application
+display_name: Webapp
```
Comment (cheshire137) [inline topics/webapp/index.md]: How about `Web app`?

File: topics/webapp/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+aliases: web-application
```
Comment (cheshire137) [inline topics/webapp/index.md]: Can you also add `web-app`, `webapps`, and `webapplication`?

File: topics/webapp/index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+display_name: Webapp
+short_description: A web application is any computer program that performs a specific function by using a web browser as its client.
+topic: webapp
+related: angular, perl, xml, html
```
Comment (cheshire137) [inline topics/webapp/index.md]: These seem like pretty general topics, not specifically related to web apps. What about [app](https://github.com/topics/app), [mobile-app](https://github.com/topics/mobile-app), [webdev](https://github.com/topics/webdev)?

File: topics/webapp/index.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+wikipedia_url: https://en.wikipedia.org/wiki/Web_application
+---
+Web appliaction is any program that is accessed over a network connection using HTTP, rather than existing on user's device. Applications like Flash games, online calculators, calendars, gmail, facebook, etc.
```
Comment (cheshire137) [inline topics/webapp/index.md]: `Web appliaction` => `A web application`

What about further describing the difference between a web app and other kinds of apps? I'm thinking the first sentence could become:

> A web application is any program that is accessed over a network connection using HTTP, usually via a [web browser](https://github.com/topics/browser). Web apps do not have to be installed on a user's device; they are run from a remote server. This differs from mobile apps, which are installed on the user's mobile device, and desktop applications, which are installed on the user's computer.

For the last sentence, how about:

> Common examples of web apps include Flash games, online calculators, calendars, Gmail, and Facebook.

File: topics/webapp/index.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+aliases: web-application, web-app, webapps, webapplication
+display_name: Web app
+short_description: A web application is any computer program that performs a specific function by using a web browser as its client.
+topic: webapp
+related: app, mobile-app, webdev
+wikipedia_url: https://en.wikipedia.org/wiki/Web_application
+---
+A web application is any program that is accessed over a network connection using HTTP, usually via a web browser. Web apps do not have to be installed on a user's device; they are run from a remote server. This differs from mobile apps, which are installed on the user's mobile device, and desktop applications, which are installed on the user's computer.Common examples of web apps include Flash games, online calculators, calendars, Gmail, and Facebook.
```
Comment (MikeMcQuaid) [inline topics/webapp/index.md]: `computer.Common` wants a wee space in between. Otherwise 👍

**Comment by MikeMcQuaid** [comment]
Thanks again @himanshu1603!

**Pull request merged**